client/src/store/features/users/usersSlice.js

```diff
@@ -6,7 +6,8 @@ import dummyWomanPic from '../../../icons/profile-picture-woman-unsplash.jpg'
 /* dummy user state */
 const initialState = {
   id: '61cdd39a5a14f24e4f2f89c7',
-  username: 'adalovelace',
+  // username: 'adalovelace',
```

Comment:
This is temporary measure, I hope we will remember when it is time to implement the functionality.